RecoLocalCalo/EcalRecProducers/plugins/AmplitudeComputationKernels.cu

```diff
@@ -348,12 +358,12 @@ namespace ecal {
 
         // store to global output values
         // FIXME: amplitudes are used in global directly
-        chi2s[idx] = chi2;
-        energies[idx] = resultAmplitudes(5);
+        chi2s[inputCh] = chi2;
+        energies[inputCh] = resultAmplitudes(5);
```

Comment:
same question here: what is the difference between `idx` and `inputCh` ?

Reply:
same as above